src/util/address.rs

```diff
@@ -502,7 +477,7 @@ impl Address {
     }
 
     /// Check whether or not the address is following Bitcoin
-    /// standardness rules.
+    /// standard rules.
```

Comment:
nit (non-merge blocking, not necessary to fix) :

I think standardness is the more used term.

Reply:
ack